packages/cli-kit/src/public/node/context/local.ts

```diff
@@ -201,10 +201,14 @@ export async function hasGit(): Promise<boolean> {
  * @param env - The environment variables from the environment of the current process.
  * @returns The CI platform info.
  */
-export function ciPlatform(env = process.env): {isCI: true; name: string} | {isCI: false; name?: undefined} {
+export function ciPlatform(
+  env = process.env,
+): {isCI: true; name: string; metadata: Metadata} | {isCI: false; name?: undefined; metadata?: undefined} {
   if (isTruthy(env.CI)) {
     let name = 'unknown'
-    if (isTruthy(env.CIRCLECI)) {
+    if (isTruthy(env.BITBUCKET_BUILD_NUMBER)) {
```

Comment:
I think this is going to fail for bitbucket unless you use `isSet`. Could you add a test to check that this behavior works with some expected bitbucket env vars?

Reply:
good catch, you are right. I've modified this and added a test with Bitbucket environment variables set.